README.md

```diff
@@ -393,6 +393,18 @@ Please note that all checks mentioned above have to pass for the change to be me
 
 History of checks performed by CI is available at [RedHatInsights / insights-results-aggregator](https://travis-ci.org/RedHatInsights/insights-results-aggregator).
 
+## Mock data for aggregator
+
+Data to be consumed by aggregator through Kafka broker is prepared in `utils/produce_insights_results/` subdirectory.
+Several types of data are available there:
+
+* `r_\*.json` - real data analyzed from test clusters
+* `result\*.json` - artifically created data
```

Comment:
Apparently there is no need to escape the *. In the rendered version the `\` are printed

Reply:
ah my bad. This is interesting as my highlighter urged me to add \. Thank you!

Reply:
should be fixed now